Features/CopyMoveDelete/CopyContent.feature

```diff
@@ -0,0 +1,46 @@
+Feature: Copy content using te role of Editor
+    In order to validate the copy action
```

Comment:
This sounds a bit off... validating a feature isn't really something a *user* wants. It's something we/you want :-)